src/scripts/devhub.zig

```diff
@@ -129,6 +129,35 @@ fn devhub_metrics(shell: *Shell, cli_args: CLIArgs) !void {
         "checksum message size max",
         "us",
     );
+    const format_time_ms = blk: {
+        timer.reset();
+
+        try shell.exec(
+            "./tigerbeetle format --cluster=0 --replica=0 --replica-count=1 datafile",
+            .{},
+        );
+
+        break :blk timer.read() / std.time.ns_per_ms;
+    };
+    const startup_time_ms = blk: {
+        timer.reset();
+
+        var process = try shell.spawn(
+            .{
+                .stdin_behavior = .Pipe,
+                .stdout_behavior = .Pipe,
+                .stderr_behavior = .Inherit,
+            },
+            "./tigerbeetle start --addresses=0 --cache-grid=8GiB datafile",
+            .{},
+        );
+        errdefer _ = process.kill() catch unreachable;
+
+        var port_buf: [std.fmt.count("{}\n", .{std.math.maxInt(u16)})]u8 = undefined;
+        _ = try process.stdout.?.readAll(&port_buf);
+
```

Comment:
Maybe also open a TCP connection to the port and write a ping message manually, just to check that the thing really is up? 

I _think_ waiting for the port to be bound is morally equivalent and should be fine, but, as a matter of principle, it still feels better to measure _directly_, rather than by proxy, and a direct measurement here is after sending a message.